### PR TITLE
bpo-39435: Fix docs for pickle.loads

### PR DESCRIPTION
**Pull request by hauntsaninja**
https://bugs.python.org/issue39435

Automerge-Triggered-By: @pitrou

### CODE REVIEW

**Comment by the-knights-who-say-ni** [comment]
Hello, and thanks for your contribution!

I'm a bot set up to make sure that the project can legally accept this contribution by verifying everyone involved has signed the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/) (CLA).

## Recognized GitHub username

We couldn't find a [bugs.python.org](https://bugs.python.org/) (b.p.o) account corresponding to the following GitHub usernames:

@hauntsaninja

This might be simply due to a missing "GitHub Name" entry in one's b.p.o account settings. This is necessary for legal reasons before we can look at this contribution. Please follow [the steps outlined in the CPython devguide](https://devguide.python.org/pullrequest/#licensing) to rectify this issue.




You can [check yourself](https://check-python-cla.herokuapp.com/) to see if the CLA has been received.

Thanks again for the contribution, we look forward to reviewing it!

**Comment by hauntsaninja** [comment]
Hello, it's my first time contributing to CPython!

Apologies if something is incorrect. `make patchcheck` complained about not having updating `Misc/ACKS`, although this is a trivial change, so sort of confused about that. Hopefully it'll recognise my CLA signature soon :-)

**Comment by pitrou** [comment]
Thanks for contributing this. However, I don't think there is anything to fix. The argument is positional only (hence the `/` after it), so its name is purely indicative. Here, `bytes_object` points to the kind of object that is accepted. By renaming it to `data`, that information is removed.

**Comment by pitrou** [comment]
I'm rejecting this PR.

**Comment by hauntsaninja** [comment]
Hello @pitrou , thanks for reviewing! 
I don't see a `/` for positional-only, all I see is a `*` for the remaining arguments being keyword-only. Note that the asterisk is escaped with a backslash in the `.rst` which could be a possible source of confusion?
As far as I can tell, the argument is not positional-only in pickle.rst on master (`.. function:: loads(bytes_object, \*, fix_imports=True, encoding="ASCII", errors="strict", buffers=None)`), the documentation at https://docs.python.org/3/library/pickle.html#pickle.loads or the behaviour at runtime:
```
>>> sys.version
'3.7.6 (default, Dec 30 2019, 19:38:26) \n[Clang 11.0.0 (clang-1100.0.33.16)]'
>>> inspect.signature(pickle.loads)
<Signature (data, *, fix_imports=True, encoding='ASCII', errors='strict')>
```

If you feel `bytes_object` is a better name than `data`, maybe we could actually document the argument as positional-only, and leave the fact that it is positional-or-keyword as undocumented.

**Comment by pitrou** [comment]
Hi @hauntsaninja, you're right, it seems I confused the `\` with a `/`. Sorry for that, and I'm reopening the PR.

**Comment by hauntsaninja** [comment]
Gentle bump :-)

File: Doc/library/pickle.rst [hunk 1]
```diff
@@ -252,10 +252,10 @@ process more convenient:
    .. versionchanged:: 3.8
       The *buffers* argument was added.
 
-.. function:: loads(bytes_object, \*, fix_imports=True, encoding="ASCII", errors="strict", buffers=None)
+.. function:: loads(data, \*, fix_imports=True, encoding="ASCII", errors="strict", buffers=None)
 
    Return the reconstituted object hierarchy of the pickled representation
-   *bytes_object* of an object.
+   *data* of an object.
```
Comment (pitrou) [inline Doc/library/pickle.rst]: Ok, but then you should mention that *data* has to be a bytes-like object.

**Review by pitrou — CHANGES_REQUESTED** [review]
Thank you. Just one comment.

**Comment by hauntsaninja** [comment]
I have made the requested changes; please review again

**Review by pitrou — APPROVED** [review]

**Comment by pitrou** [comment]
Thank you @hauntsaninja !

**Pull request merged**